Moblin/Various/Model.swift

```diff
@@ -4955,7 +4970,7 @@ extension Model {
     }
 
     func updateRemoteControlAssistantStatus() {
-        guard showingRemoteControl, remoteControlAssistant?.isConnected() == true else {
```

Comment:
I would prefer the remote control assistant to only fetch information from the remote control streamer when the assistant is actually in use, to minimize the network traffic from the streamer.

Reply:
ok reverted it